src/components/SearchBar/SearchBar.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router-dom';
+import SearchBar from '../SearchBar/SearchBar';
+import SearchBy from '../SearchBy/SearchBy';
+
+describe('< SearchBar />', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<SearchBar />);
+  });
+
+  it('should be rendered', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should have correct state', () => {
+    expect(wrapper.state().searchBy).toEqual('title');
```

Comment:
You should not test this. Because it is gonna be true all the time. You change state directly here and check that it has been changed. You can be sure, enzyme will rerender component and set new state. So, remove this test case